### DISCUSSION TITLE
Xcode15 build  ios app run on IOS17 device encount error:   VM - (arg = 0x3) mach_vm_allocate_kernel failed within call to vm_map_enter SIGNAL 6 Abort trap: 6

### DISCUSSION THREAD

**Discussion by lin20260407** [community/community#113753] [Mobile]
### Select Topic Area

Bug

SIGNAL 6 Abort trap: 6

### Body

Has anyone ever had that happen to them? I'd like to know.

**Comment by Qacckell** [comment]
Just bonnie and clyde

**Comment by klffah** [comment]
- [ ] # []()**![1711320682681.jpg](https://github.com/community/community/assets/143175556/65b20e89-1769-4f37-bd03-f12e5e720032)

**

**Comment by esahmed21** [comment]
Hi @weitsunglin 👋 

It seems like you're encountering an issue with building and running an iOS app using Xcode 15 on an iOS 17 device. However this discussion might not be the appropriate place to seek assistance for this issue. This discussion is only related for the Github Mobile app. So maybe you can try Stack Overflow or other communities like the Apple Developer Forums would likely be more helpful. 

Thanks for getting in touch.
